packages/slice-machine/lib/utils/version.ts

```diff
@@ -0,0 +1,49 @@
+import path from "path";
+import semver from "semver";
+import fs from "fs";
+import fetch, { Response } from "node-fetch";
+import {
+  name as packageName,
+  version as installedVersion,
+} from "../../package.json";
+
+export class HTTPResponseError extends Error {
```

Comment:
We don't have already a type for Error ????

Reply:
Nope, it got removed during review